src/components/progress-report/workflow/events/workflow-updated.event.ts

```diff
@@ -0,0 +1,14 @@
+import { ID, Session, UnsecuredDto } from '~/common';
+import { ProgressReportStatus as Status } from '../../dto';
+import { ProgressReportWorkflowEvent as WorkflowEvent } from '../dto/workflow-event.dto';
+import { InternalTransition } from '../transitions';
+
+export class WorkflowUpdatedEvent {
+  constructor(
+    readonly reportId: ID,
+    readonly previousStatus: Status,
+    readonly next: InternalTransition | Status,
```

Comment:
```suggestion
```
Grab from 
```ts
workflowEvent.transition
workflowEvent.status
```

Reply:
Which is why `status` is also available.

Reply:
Since `next` contained the actual transition object which contains the `notify` property, I'm left with only a transitionId inside the workflowEvent, `workflowEvent.transition`. Which API (or other method) should I now use to obtain a transition object by transitionId in order to obtain the notify property in order to get the valid team member roles to notify.

Reply:
Mmm yes you're right. I apologize. It could be possible to use the normal secured version of WorkflowEvent here which would have the transitions object. Or you could look it up easily like [this](https://github.com/SeedCompany/cord-api-v3/blob/3793f29ef7598c8cc9a075827d6c082995e119b9/src/components/progress-report/workflow/progress-report-workflow.service.ts#L46-L46). However, I probably wouldn't recommend those over what you have now.

Feel free to leave this as is 💪